packages/story-editor/src/types/currentUserProvider.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Internal dependencies
+ */
+import type { User } from './configProvider';
+
+export interface updateCurrentUserProps {
```

Comment:
This type can just live in `currentUserProvider.tsx`.

Also, let's capitalize the interface name for consistency.

Reply:
This type is used in this file.

Reply:
Argh, right.

Let's capitalize it still.